package.json

```diff
@@ -27,7 +27,7 @@
   },
   "scripts": {
     "start": "yarn build && yarn serve",
-    "build": "yarn build-scss && yarn build-js",
+    "build": "yarn build-scss && yarn build-js && yarn create-class-references",
```

Comment:
It's needed for the website, but not for Vanilla build, so probably can be moved out from here and part of 'start'?